html/index.html

```diff
@@ -0,0 +1,154 @@
+        <p><a class="change" href="https://www.britannica.com/place/Istanbul">Here</a> you can read more if you are interested!</p>
+        </div>
+      
+      <div class="img-3">
+        <div class="caption">
+        <span id="visit" class="transparent">PLACES TO VISIT</span>
+        </div>
+      </div>
+      
+      <div class="rel">
+        <p>This is an information which you can easily find on the internet. For example <a class="change" href="https://www.lonelyplanet.com/turkey/istanbul">here</a> you can find the places to visit as a tourist.</p>
+        <p>However, the city has a population of 15.07 million on a total area of 5,343 square kilometers. Basically this means: it is very crowded. There are people everywhere you go. After a few hours, you will prbably be feeling exhausted because of seeing so many people wherever you go. So, here are my tips for people who wants to visit at least relatively peaceful places.  
+            <ul>
+<li>Visit the hidden streets of Cihangir.</li>
+<li>Visit the islands in the weekdays and never at the weekends!</li>
+<li>Don't go to Taksim, there is nothing to see there.</li>
+<li>Visit Agva, Kefken, Sile</li>
+
+
+            </ul>
+        </p>
+        </div>
+      
+      <div class="img-4">
+        <div class="caption">
+       <span id="food" class="transparent">FOOD & DRINK</span>
+    </div>
+</div>
+
+       <div class="rel">
+       <p>Unlike European cities, you can find something to eat "literally" at every corner. However, you can not find tasty food everywhere. I will give you a few recommandations.
+           <ul>
+<li>If you want a fantastic Bosphorus view with good drink and snacks, definetely go to <a class="change" href="http://www.5kat.com/tr" target="_blank">Besinci Kat</a>.</li>
+<li><a class="change" href="http://www.ciya.com.tr/" target="_blank">Ciya</a> is very traditional and authentic</li>
+<li><a class="change" href="http://www.voguerestaurant.com/" target="_blank">Vogue</a> is definetely a must try.</li>
+
+
+
+           </ul>
+       </p>
+       </div>
+        
+       <div class="img-5">
+            <div class="caption">
+           <span id="street" class="transparent">STREETS</span>
+        </div>
+    </div>
+        
+    <div class="rel">
+            <p>To see good examples of Istanbul streets, walk at Besiktas. Go to every small street even the ones at the edges. Istanbul is not flat and there are many ramps on the streets. With ramp I really mean it. If you are not trained enough, you can be breathless even after 1 ramp. THey are reaaaalllly rough. However, the streets are full of surprises. There are good and bad surprises in the streets of Istanbul. You can never know what you will experience once you step out of your house. So, be prepared for everything and get ready for a real experience!</p>
+            </div>
+
+<div class="img-6">
```

Comment:
try to avoid numbers in classes. either a generic class name or an id if you wan to be specific. that's just good practice =)